docs/source/using/profiles.adoc

```diff
@@ -15,46 +15,52 @@ A profile is an instance of the Minishift VM along with all of its configuration
 The profile feature allows you to create and manage these isolated instances of Minishift.
 
 Each {project} profile is created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
-See xref:../using/basic-usage.adoc#environment-variables[environment variables], if you want to apply some common configurations (CPU, memory etc) to all profiles.
+Refer to the use of xref:../using/basic-usage.adoc#environment-variables[environment variables], if you want to make sure that certain configuration settings, for example `cpus` or `memory`, get applied to all profiles.
```

Comment:
I like this rewrite. Just need to remove the comma after "environment variables".

Reply:
ok